code/datums/supplypacks/hospitality_vr.dm

```diff
@@ -0,0 +1,143 @@
+
+/datum/supply_packs/randomised/hospitality/bakery_vr
+	num_contained = 5
+	contains = list(
+			/obj/item/weapon/reagent_containers/food/snacks/baguette,
+			/obj/item/weapon/reagent_containers/food/snacks/appletart,
+			/obj/item/weapon/reagent_containers/food/snacks/berrymuffin,
+			/obj/item/weapon/reagent_containers/food/snacks/bunbun,
+			/obj/item/weapon/reagent_containers/food/snacks/cherrypie,
+			/obj/item/weapon/reagent_containers/food/snacks/cookie,
+			/obj/item/weapon/reagent_containers/food/snacks/croissant,
+			/obj/item/weapon/reagent_containers/food/snacks/donut/normal,
+			/obj/item/weapon/reagent_containers/food/snacks/donut/jelly,
+			/obj/item/weapon/reagent_containers/food/snacks/donut/cherryjelly,
+			/obj/item/weapon/reagent_containers/food/snacks/muffin,
+			/obj/item/weapon/reagent_containers/food/snacks/pie,
+			/obj/item/weapon/reagent_containers/food/snacks/plump_pie,
+			/obj/item/weapon/reagent_containers/food/snacks/plumphelmetbiscuit,
+			/obj/item/weapon/reagent_containers/food/snacks/poppypretzel,
+			/obj/item/weapon/reagent_containers/food/snacks/sugarcookie,
+			/obj/item/weapon/reagent_containers/food/snacks/waffles
+			)
+	name = "Bakery products crate"
+	cost = 15
+	containertype = /obj/structure/closet/crate/freezer
+	containername = "Bakery products crate"
+
+/datum/supply_packs/randomised/hospitality/cakes_vr
+	num_contained = 5
+	contains = list(
+			/obj/item/weapon/reagent_containers/food/snacks/sliceable/applecake,
+			/obj/item/weapon/reagent_containers/food/snacks/sliceable/birthdaycake,
+			/obj/item/weapon/reagent_containers/food/snacks/sliceable/carrotcake,
+			/obj/item/weapon/reagent_containers/food/snacks/sliceable/cheesecake,
+			/obj/item/weapon/reagent_containers/food/snacks/sliceable/chocolatecake,
+			/obj/item/weapon/reagent_containers/food/snacks/sliceable/lemoncake,
+			/obj/item/weapon/reagent_containers/food/snacks/sliceable/limecake,
+			/obj/item/weapon/reagent_containers/food/snacks/sliceable/orangecake,
+			/obj/item/weapon/reagent_containers/food/snacks/sliceable/plaincake
+			)
+	name = "Cake crate"
+	cost = 50
+	containertype = /obj/structure/closet/crate/freezer
+	containername = "Cake crate"
+
+/datum/supply_packs/randomised/hospitality/mexican_vr
+	num_contained = 5
+	contains = list(
+			/obj/item/weapon/reagent_containers/food/snacks/cheeseburrito,
+			/obj/item/weapon/reagent_containers/food/snacks/cheesenachos,
+			/obj/item/weapon/reagent_containers/food/snacks/enchiladas,
+			/obj/item/weapon/reagent_containers/food/snacks/fuegoburrito,
+			/obj/item/weapon/reagent_containers/food/snacks/meatburrito,
+			/obj/item/weapon/reagent_containers/food/snacks/nachos,
+			/obj/item/weapon/reagent_containers/food/snacks/taco
+			)
+	name = "Mexican food crate"
+	cost = 15
+	containertype = /obj/structure/closet/crate/freezer
+	containername = "Mexican food crate"
+
+/datum/supply_packs/randomised/hospitality/asian_vr
+	num_contained = 5
+	contains = list(
```

Comment:
This crate can stay but make it more expensive and remove everything except `generalschicken` and `hotandsoursoup` and we can lore this off as Chinese Takeout crate.